src/graph/resolvers/campaign.js

```diff
@@ -20,6 +20,12 @@ const getNotifyDefaults = async (advertiserId, user) => {
   return notify;
 };
 
+const updateNotifications = async (campaignId) => {
+  contactNotifier.scheduleCampaignCreated({ campaignId });
+  contactNotifier.scheduleCampaignStarted({ campaignId });
```

Comment:
Also, have a look at the `each` function within the `async` package. These are probably a good candidates for running in parallel.
https://caolan.github.io/async/docs.html#each